test/jdk/java/nio/channels/FileChannel/LargeMapTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.lang.foreign.MemorySegment;
+import java.lang.foreign.MemorySession;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.file.Path;
+import static java.nio.file.StandardOpenOption.*;
+
+/*
+ * @test
+ * @enablePreview
+ * @bug 8286637
+ * @requires os.family == "windows"
```

Comment:
What was the conclusion on whether this test should run everywhere?

Reply:
There wasn't one exactly. I am currently testing on all main platforms with 20 repeats. If that succeeds perhaps it should be allowed to run on all.

Reply:
20 test repeats passed on all main platforms in the CI.

Reply:
Good. You might want to check the execution times in case the test needs to run with /timeout set to some large value. I think we've had to do that a few other tests that create multi GB files.